src/DynamoCore/Graph/ModelBase.cs

```diff
@@ -272,23 +272,23 @@ public bool UpdateValue(UpdateValueParams updateValueParams)
         }
 
         /// <summary>
-        /// This method is currently used as a way to send an event to ModelBase 
-        /// derived objects. Its primary use is in DynamoNodeButton class, which 
+        /// This method is currently used as a way to send an event to ModelBase
+        /// derived objects. Its primary use is in DynamoNodeButton class, which
         /// sends this event when clicked.
         /// </summary>
         /// <param name="eventName">The name of the event.</param>
         /// <param name="recorder"></param>
         /// <returns>Returns true if the call has been handled, or false otherwise.
         /// </returns>
-        internal bool HandleModelEvent(string eventName, UndoRedoRecorder recorder)
+        internal bool HandleModelEvent(string eventName, Int32 value, UndoRedoRecorder recorder)
```

Comment:
I notice that the docs comment for this function says 

> Its primary use is in DynamoNodeButton class, which sends this event when clicked.

Is this an incorrect comment?  I can't find any evidence to support that this is true.